proxywasm/internal/abi_callback_version.go

```diff
@@ -16,7 +16,7 @@ package internal
 
 import "time"
 
-//export proxy_abi_version_0_2_0
+//go:wasmexport proxy_abi_version_0_2_0
```

Comment:
Can we target 0_2_1? (the current latest ABI)
https://github.com/proxy-wasm/spec/tree/main/abi-versions

Reply:
I would prefer to make this change in a separate PR to minimize functional changes in this PR.

Reply:
SGTM.